docs/SDL/OnReceivedPolicyUpdate/assets/SDL.OnReceivedPolicyUpdateEXTERNAL_PROPRIETARY.txt

```diff
@@ -1,35 +1,36 @@
--- Policy Update Sequence
-title Policy Table Update (EXTERNAL_PROPRIETARY)
-
-participant Policy Server
-participant Mobile
-participant SDL
-participant HMI
-
-note over SDL: PT update is required
-
-SDL->HMI: SDL.OnStatusUpdate(UPDATE_NEEDED)
-SDL->SDL: Creates PT snapshot
-SDL->HMI: BC.PolicyUpdate(path_to_PTS)
-HMI-->SDL: SUCCESS: BC.PolicyUpdate
-HMI->SDL: SDL.GetPolicyConfigurationData\n(policy_type=module_config, property=endpoints)
-note over SDL: Get endpoints from policies database
-SDL-->HMI: SDL.GetPolicyConfigurationData(value)
-note over HMI: Parse `url` for `custom_vehicle_data_mapping_url`
-HMI->HMI: PT snapshot\nencryption
-HMI->SDL: BC.OnSystemRequest(url, path_to_PTS)
-SDL->HMI: SDL.OnStatusUpdate(UPDATING)
-SDL->Mobile: OnSystemRequest(url, PTS_in_binaryData)
-Mobile->Policy Server: PT snapshot (PTS)
-Policy Server->Mobile: PT update (PTU)
-Mobile->SDL: SystemRequest(PTU_in_binaryData)
-SDL-->Mobile: SUCCESS: SystemRequest
-SDL->HMI: BC.SystemRequest(path_to_PTU)
-HMI-->SDL: SUCCESS: BC.SystemRequest
-HMI->HMI: PT update\ndecryption
-HMI->SDL: SDL.OnReceivedPolicyUpdate(path_to_PTU)
-SDL->SDL: Validate PTU
-SDL->SDL: Merge PTU into Local PT
-SDL->HMI: SDL.OnStatusUpdate(UP_TO_DATE)
-SDL->HMI: BC.DecryptCertificate(path_to_CRT)
-HMI-->SDL: SUCCESS: BC.DecryptCertificate
\ No newline at end of file
+  
+-- Policy Update Sequence
+title Policy Table Update (EXTERNAL_PROPRIETARY)
+
+participant Policy Server
+participant Mobile
+participant SDL
+participant HMI
+==<align:center>SDL is built with with **"-DEXTENDED_POLICY: EXTERNAL_PROPRIETARY"** flag\ndevice is consented==
+note over SDL: PT update is required
+
+SDL->HMI: SDL.OnStatusUpdate(UPDATE_NEEDED)
+SDL->SDL: Creates PT snapshot
+SDL->HMI: BC.PolicyUpdate(path_to_PTS)
+HMI-->SDL: SUCCESS: BC.PolicyUpdate
+SDL->HMI: SDL.OnStatusUpdate(UPDATING)
+SDL->SDL: Starts timeout taken from "timeout_after_x_seconds" of Local PT
+HMI->SDL: SDL.GetPolicyConfigurationData\n(policy_type=module_config, property=endpoints)
+note over SDL: Get endpoints from policies database
+SDL-->HMI: SDL.GetPolicyConfigurationData(value)

```

Comment:
Instead of just having `value` we could have
```suggestion
SDL-->HMI: SDL.GetPolicyConfigurationData\n(["{\"0x07\":{\"default\":[\"http://x.x.x.x:3000/api/1/policies\"]}, ...}"])
```
or something like that just to give the reader a better idea as to what type of data will be returned by the `SDL.GetPolicyConfigurationData` RPC